client/state/index.js

```diff
@@ -71,6 +72,7 @@ export const reducer = combineReducers( {
 	componentsUsageStats,
 	countryStates,
 	currentUser,
+	directly,
```

Comment:
I wouldn't put it in `ui/directly` using `ui/olark` as a reference. Some of the stuff in `ui/olark` is in ~the wrong~ a less than ideal place and will be ripped out later this month.